devimint/src/devfed.rs

```diff
@@ -136,7 +136,26 @@ impl DevJitFed {
             let bitcoind = bitcoind.clone();
             || async move {
                 let bitcoind = bitcoind.get_try().await?.deref().clone();
-                Ok(Arc::new(Esplora::new(&process_mgr, bitcoind).await?))
+                let esplora = Esplora::new(&process_mgr, bitcoind).await?;
+
+                let client = esplora_client::Builder::new(&format!(
+                    "http://localhost:{}",
+                    process_mgr.globals.FM_PORT_ESPLORA
+                ))
+                .build_async()
+                .expect("esplora client build failed");
+
+                poll("esplora server ready", || async {
+                    client
+                        .get_fee_estimates()
+                        .await
+                        .map_err(|e| ControlFlow::Continue(anyhow::anyhow!(e)))?;
+
+                    Ok(())
+                })
+                .await?;
```

Comment:
Nit: Could be a method on something or a standalone function.

Reply:
Done.